assets/js/pages/Profile/ProfileForm.jsx

```diff
@@ -180,6 +184,38 @@ function ProfileForm({
               disabled
             />
           </div>
+          {analyticsEnabledConfig && (
```

Comment:
I think you can wrap both entries in the sama condition:
```
{analyticsEnabledConfig && (
<>
<Label className="col-start-1 col-span-2 pt-2">
    Analytics Opt-in
</Label>
<div className="col-start-3 col-span-4">
              <div className="pt-2">
...
</>
)}
```

I mean, you don't need 2 "if" conditions

Reply:
Thanks for this. I was wondering how to do this with using a empty div.